db-client-java/src/test/java/com/eventstore/dbclient/samples/quick_start/QuickStart.java

```diff
@@ -43,10 +42,22 @@ public static void Run() throws ParseError, ExecutionException, InterruptedExcep
                 .forwards()
                 .fromStart();
 
-        ReadResult result = client.readStream("some-stream", 10, options)
-                .get();
+        client.readStream("some-stream", 10, options).subscribe(new ReadSubscriber() {

```

Comment:
**SUGGESTION:** This sample intent is to show how to read a set of events from the stream, so it should be more like the one above: https://github.com/EventStore/EventStoreDB-Client-Java/pull/106/files#diff-c53c867ba6cee9fc84ea5583415293244d6371732798e517b74de2fe8523e770R96